### PR TITLE
make input button useful

### PR DESCRIPTION
**Pull request by Hikarun15**
changed the color of the input button and add opacity to it

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -120,7 +120,7 @@ textarea {
   border-radius: 5px;
   border: 0;
   color: white;
```
Comment (oladhari) [inline css/styles.css]: please use only one kind of value for colors, use hex, or rgba or hardcoded color (orange, yellow, white)
using all in the same file will make it hard to maintain in the future

Comment (morinagahikaru) [reply css/styles.css]: OK!I integrated kinds of value for colors!

Comment (Hikarun15) [reply css/styles.css]: I gave those colors the  same kind of value!

**Review by oladhari — APPROVED** [review]
LGTM
thank you

**Comment by Hikarun15** [comment]
Can I merge this branch?

**Pull request merged**